src/cmap/connection.ts

```diff
@@ -401,22 +401,14 @@ export class Connection extends TypedEventEmitter<ConnectionEvents> {
         this.emit(Connection.CLUSTER_TIME_RECEIVED, document.$clusterTime);
       }
 
-      if (operationDescription.command) {
```

Comment:
Should we finish this removal? .command shouldn't be needed on operation descriptions, along with the CommandOptions.command boolean that sets it, its hardcoded to true in Connection.command

Reply:
yeah, good call.  done!